index.html

```diff
@@ -20,7 +20,8 @@
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/skeleton.css">
   <link rel="stylesheet" href="css/style.css">
-
+  <noscript>
+    <link rel="stylesheet" href="css/noscript.css"></noscript>
```

Comment:
put this either all on the same line, or put the end tag at the same indentation level as the start